### ISSUE TITLE
Detail view is not uniform

### ISSUE THREAD

**Issue by FiedlerJan** [hochschule-darmstadt/einander-helfen#126]
**Describe the bug**
If the task text is short, the spacings on the detail view do not match the spacing of the other posts.

**To Reproduce**
1. Go to 'https://cai-einander-helfen.fbi.h-da.de/posts?q=&location=&radius=&page=1'
2. Click on post '10.000 Kirchenvorstandmitglieder gesucht'
3. Click on post '2. Geschäftsführer/in'
4. The spacings in the detail view do not match

**Expected behavior**
The spacings are equal on all posts

**Screenshots**
![grafik](https://user-images.githubusercontent.com/64203299/86293115-4efe7480-bbf2-11ea-9f18-f2074b105970.png)

![grafik](https://user-images.githubusercontent.com/64203299/86293138-59b90980-bbf2-11ea-8ae0-7ff91e280cdb.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox
 - Version 77

**Smartphone (please complete the following information):**
Not used.

**Additional context**
None

**Comment by BaderChouaib** [comment]
Issue seems to only occur on large and extra large devices. May have something to do with available space in detail view
Same behaviour on these posts:
https://cai-einander-helfen-staging.fbi.h-da.de/posts/f61d4a2586044dd3cb2be60c7ee08a39?q=Beisitzer%20im%20Vorstand%20f%C3%BCr%20unseren%20Verein%20gesucht&location=&radius=&page=1

https://cai-einander-helfen-staging.fbi.h-da.de/posts/140fe53efb2657043ac87725d0af3e39?q=Zumba&location=&radius=&page=1

https://cai-einander-helfen.fbi.h-da.de/posts/8d6679043e84104fe3181fa233a4b568?q=&location=&radius=&page=181

In fact the spacings on those posts are similar on same larger devices+. The bug seems to occur when there is only max. one line of text in the task field. Might more complex but thats what it seems

**Comment by sgrobarek** [comment]
Maybe I have found a solution. I changed the v-simple-table to a div and added a "padding right" to the categories. I also changed the color to black and if you look at the posts 1 and 2 on the second page, it seems to work compare to the staging version.
![Screen Shot 2020-07-07 at 04 53 12](https://user-images.githubusercontent.com/64202529/86754627-063e2580-c00f-11ea-93ee-addda44d9b13.png)
![Screen Shot 2020-07-07 at 04 53 16](https://user-images.githubusercontent.com/64202529/86754656-0d653380-c00f-11ea-8953-1f6af7c35389.png)

**Comment by jschurse** [comment]
Is this still working with different resolutions?

sgrobarek <notifications@github.com> schrieb am Di., 7. Juli 2020, 11:03:

**Comment by sgrobarek** [comment]
Yes, it seems to work with different resolutions. I tried 4k, 1080p and 720p. On mobile view we can use the v-simple-table, because I copied in issue #76 the "table" for the mobile view to get the detail view under the selected post.

**Comment by xcaverxiii** [comment]
@KaiDrieHda @DianaPankratz  please check if this is use still an issue.

**Comment by DianaPankratz** [comment]
@xcaverxiii yes this bug still exists. For example search for the location "Maierhöfen" and comapre the posts "Gemeinde Gestratz - Bauhof" and "Westallgäuer Waldkindergarten"
